### DISCUSSION TITLE
Does Fooocus support Automatic1111 prompts attention tweaks

### DISCUSSION THREAD

**Discussion by adumont** [lllyasviel/Fooocus#1901] [Q&A] [answered]
Does Fooocus support any, some, all the syntax Automatic1111 prompts attention tweaks (related to changing the prompts after some steps):

- "(x)": emphasis. Multiplies the attention to x by 1.1. Equivalent to (x:1.1)
- "[x]": de-emphasis, divides the attention to x by 1.1. Approximate to (x:0.91) (Actually 0.909090909...)
- "(x:number)": emphasis if number > 1, deemphasis if < 1. Multiply the attention by number.
- "\(x\)": Escapes the parentheses, this is how you'd use parenthesis without it causing the parser to add emphasis.
- "[x:number]": Ignores x until number steps have finished. (People sometimes think this does de-emphasis, but it does not)
- "[x::number]": Ignores x after number steps have finished.
- "[x:x:number]": Uses the first x until number steps have finished, then uses the second x.
- "[x|x]", "[x|x|x]", etc. Alternates between the x's each step.

Reference: https://www.reddit.com/r/StableDiffusionInfo/comments/ylp6ep/some_detailed_notes_on_automatic1111_prompts_as/
Tutorial: https://www.youtube.com/watch?v=MgLpmksylOc

**Comment by poor7** [comment, marked as answer]
Works only: 1) (x:1,1)  2) (x:0,91) и 4) (x)

**Reply by adumont** [reply]
Thanks. I did know about those parens tweaks. I'm particularly interested in the bracket tweaks, too bad it's not supported:

- "[x:number]": Ignores x until number steps have finished. (People sometimes think this does de-emphasis, but it does not)
- "[x::number]": Ignores x after number steps have finished.
- "[x:x:number]": Uses the first x until number steps have finished, then uses the second x.
- "[x|x]", "[x|x|x]", etc. Alternates between the x's each step.
